packages/rollup-plugin/src/index.ts

```diff
@@ -0,0 +1,96 @@
+import type { Plugin } from 'rollup';
+import {
+  cssFileFilter,
+  processVanillaFile,
+  compile,
+  IdentifierOption,
+  getSourceFromVirtualCssFile,
+  serializeCss,
+} from '@vanilla-extract/integration';
+import { relative, normalize, dirname } from 'path';
+
+const virtualPrefix = 'virtual:vanilla-extract:';
+const vanillaSuffix = '.vanilla.css';
+
+interface Options {
+  identifiers?: IdentifierOption;
+  cwd?: string;
```

Comment:
Most other bundlers have a built-in idea of project root or cwd. Does rollup not?

Reply:
I could not find anything related to this. But it seems to work fine with `process.cwd`. Not sure this option is needed.